api/src/pcapi/core/mails/transactional/bookings/booking_confirmation_to_beneficiary.py

```diff
@@ -105,5 +105,6 @@ def get_booking_confirmation_to_beneficiary_email_data(
             "BOOKING_LINK": booking_app_link(booking),
             "OFFER_WITHDRAWAL_DELAY": bookings_common.get_offer_withdrawal_delay(booking),
             "OFFER_WITHDRAWAL_TYPE": bookings_common.get_offer_withdrawal_type(booking),
+            "FEATURES": ", ".join(stock.features),  # type: ignore [arg-type]
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Pourquoi on se retrouve avec une erreur mypy ici ?